src/sdx_pce/topology/manager.py

```diff
@@ -90,24 +99,36 @@ def add_topology(self, data):
                 )
 
             # Nodes
-            nodes = topology.get_nodes()
```

Comment:
Historical reason-:) I had some manually c&p setters/getters, mixed with annotated properties.. 
So I use this opportunity to replace and make them consistent.
So in both sdx-lc and sdx-controller, we need to retest in case something not covered by unittest and uses old calls.